File: Server/CTSE-Exam-In-App/DTOs/Question/QuestionResponse.cs
using System.Collections.Generic;
using CTSE_Exam_In_App.DTOs.Base;

namespace CTSE_Exam_In_App.DTOs.Question
{
    public class QuestionResponse : BaseResponse
    {
        public IEnumerable<Models.Question> Questions { get; set; }
    }
}

File: Client/Android/app/src/main/java/com/example/examinapp/dataaccess/dtos/question/QuestionResponse.java
package com.example.examinapp.dataaccess.dtos.question;

import com.example.examinapp.dataaccess.dtos.base.BaseResponse;
import com.example.examinapp.models.QuestionModel;

import java.util.Date;
import java.util.List;

public class QuestionResponse extends BaseResponse {

    private List<QuestionModel> questions;

    public QuestionResponse(boolean isSuccess, int status, String message, Date respondDateTime, List<QuestionModel> questions) {
        super(isSuccess, status, message, respondDateTime);
        this.questions = questions;
    }

    public List<QuestionModel> getQuestions() {
        return questions;
    }

    public void setQuestions(List<QuestionModel> questions) {
        this.questions = questions;
    }
}
